apps/dashboard/app/views/layouts/filenavigator.html.erb

```diff
@@ -0,0 +1,7 @@
+<%
+content_for :head do
+  javascript_pack_tag 'files/filenavigator'
+end
+%>
+
+<%= render template: "layouts/modal", locals: { layout_container_class: 'container-fluid' } %>

```

Comment:
Do we need new layouts? I thought this was going to be a modal on an existing page?